labels.json

```diff
@@ -0,0 +1,119 @@
+[
+  {
+    "name": "addition/proposal",
+    "description": "New features or enhancements",
+    "color": "d4c5f9"
+  },
+  {
+    "name": "anchor permanence",
+    "description": "Identifies issues opened by other standards organizations, as per WHATWG's [anchor permanence policy](https://whatwg.org/working-mode#anchors)",
+    "color": "ececec",
+    "url_exclude_is_open": true
+  },
+  {
+    "name": "clarification",
+    "description": "Standard could be clearer",
+    "color": "fef2c0"
+  },
+  {
+    "name": "compat",
+    "description": "Standard is not web compatible",
```

Comment:
Sometimes we use this for cases where something needs to be standardized so browsers can become web compatible, I think?